kernel/yosys.cc

```diff
@@ -149,16 +149,18 @@ void yosys_banner()
 	log("\n");
 }
 
-int ceil_log2(int x)
+int ceil_log2(const int x)
 {
+#if defined(__GNUC__)
+        return (x > 0) * (32 - __builtin_clz(x-1) - (x == 1));
```

Comment:
``return x > 1 ? (8*sizeof(int)) - my__builtin_clz(x-1) : 0;``

Reply:
Did you mean to put "my__builtin_clz"?

Reply:
Done!

Reply:
No, I did not. That was just a left-over from the correctness proof.  ;)